src/PowerShell/Microsoft.WinGet.DSC/Microsoft.WinGet.DSC.psm1

```diff
@@ -74,9 +74,9 @@ class WinGetUserSettings
     # Gets the current UserSettings by looking at the settings.json file for the current user.
     [WinGetUserSettings] Get()
     {
-        Assert-WinGetCommand "Get-WinGetUserSettings"
+        Assert-WinGetCommand "Get-WinGetUserSetting"

```

Comment:
I removed these asserts in a change that has been merged recently, as well as other significant changes to this file that are not reflected here.  I'm not sure exactly what has happened, but we will have to work out the conflicts here before it can be merged.

Reply:
I will try to resolve it after I check on my own PR, but I'm not sure if I can update this PR directly.

Reply:
Nope, I'm too deep in the changes... I removed these in my own PR that has not been merged yet.